scripts/filter_bats_list_by_file.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This script is not being used at the moment correct?

Reply:
no, this file isn't being used.